relayer/relays/beacon/state/beacon.go

```diff
@@ -164,7 +164,8 @@ type BeaconState interface {
 	GetBlockRoots() [][]byte
 	GetTree() (*ssz.Node, error)
 	GetFinalizedCheckpoint() *Checkpoint
-	GetSyncSyncCommittee() *SyncCommittee
+	GetCurrentSyncCommittee() *SyncCommittee
+	GetNextSyncCommittee() *SyncCommittee
```

Comment:
We could yes, but that would make the initial checkpoint payload massive, which I think could cause more problems that benefits. For example, the [message queue pallet config message size](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs#L367) may have to be increased again. We don't send 2 sync committees anywhere else either so I'm not sure it would make sense to allow such a large payload.